content/blog/adoptopenjdk-16-available/index.md

```diff
@@ -0,0 +1,52 @@
+* [JEP 390 brought warnings for value-based classes](https://openjdk.java.net/jeps/390).
+* [With JEP 392, jpackage, a tool to create native application packages like an .exe, was promoted to a production feature](https://openjdk.java.net/jeps/392).
+* [JEP 393 started a third incubation phase of the Foreign-Memory Access API that replaces Unsafe, and more](https://openjdk.java.net/jeps/393).
+* [JEP 394 adds pattern matching for instanceof](https://openjdk.java.net/jeps/394).
+* [With JEP 395, Records are now production-ready](https://openjdk.java.net/jeps/395).
+* [JEP 396 encapsulates additional JDK internals by default](https://openjdk.java.net/jeps/396).
+* [JEP 397 started the second preview of sealed classes](https://openjdk.java.net/jeps/397).
+
+For a complete list of the enhancements (including ones that only impact developers of OpenJDK), [see the JDK 16 overview over at OpenJDK](http://openjdk.java.net/projects/jdk/16/).
+
+### AdoptOpenJDK for Alpine Linux With musl libc 
+
+[Alpine Linux](https://alpinelinux.org/) is a popular Linux distribution for container workloads because of its small footprint. Contrary to most other Linux distributions, Alpine Linux is not based on the [C library created by the GNU project](https://www.gnu.org/software/libc/) (usually referred to as "glibc") but uses [musl libc](http://musl.libc.org) instead. So far, OpenJDK has not supported musl libc but only glibc. Therefore, we had to add the GNU C library to our container images based on Alpine Linux, which increased the container images' size. With AdoptOpenJDK 16, this is no longer necessary. We now have separate variants of AdoptOpenJDK 16 that are purpose-built for musl libc that can be downloaded as a tarball and are also available as ready-made container images.  
+
+### Preview of AdoptOpenJDK for Windows on ARM<a name="adoptopenjdk-for-windows-on-arm"></a>
+
+Our friends from Microsoft, led by our very own [Martijn Verburg](https://twitter.com/karianna), have ported [OpenJDK to Windows on ARM](https://openjdk.java.net/jeps/388) so that you can run Java (and Java-based applications) natively on your [Surface Pro X](https://www.microsoft.com/en-us/p/surface-pro-x/8qg3bmrhnwhk) or the Lenovo IdeaPad 5G. Therefore, AdoptOpenJDK 16 is now available for Windows on ARM, too, but only as a preview. The reason is that Windows on ARM is a relatively new thing with minimal hardware and software availability. Therefore, we have not yet been able to test it on a regular schedule like we usually do.
+
+### Removal of 10 Symantec Root Certificates
+
+As part of the general distrust of Symantec root certificates, 10 of them have been removed from AdoptOpenJDK. For details, see [the related issue over at the Mozilla Foundation](https://bugzilla.mozilla.org/show_bug.cgi?id=1670769).
+
+### Removal of XL Variants of OpenJ9
```

Comment:
Would this be better phrased as "Merging of the OpenJ9 standard and large (XL) variants"? We haven't removed any functionality (which the heading might imply), just the separate download for it.